src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/AddIncomeCommandParserTest.java

```diff
@@ -59,9 +63,17 @@ public void parse_allFieldsPresent_success() {
     @Test
     public void parse_optionalFieldsMissing_success() {
         // zero categories
-        Income expectedIncome = new TransactionBuilder(BUBBLE_TEA_2).withCategories().buildIncome();
+        Income expectedIncomeWithNoCategories = new TransactionBuilder(BUBBLE_TEA_2).withCategories()
```

Comment:
```suggestion
        Income expectedIncomeWithNoCategories = new TransactionBuilder(BUBBLE_TEA_2)
```

Redundant method call.